src/config/backpack/base.php

```diff
@@ -68,11 +68,19 @@
 
     /*
     |--------------------------------------------------------------------------
-    | User Model
+    | Authentication
     |--------------------------------------------------------------------------
     */
 
     // Fully qualified namespace of the User model
     'user_model_fqn' => '\App\User',
```

Comment:
Right... that might be very confusing for someone who wants to start using the feature, but doesn't have the views, you're right. Let's call it a breaking change then.